Widgets/Widgets/PictureOfTheDayWidget.swift

```diff
@@ -33,37 +33,20 @@ final class PictureOfTheDayData {
     // MARK: Public
 
     func fetchLatestAvailablePictureEntry(for imageSize: CGSize, usingCache: Bool = false, completion userCompletion: @escaping (PictureOfTheDayEntry) -> Void) {
-        WidgetController.shared.startWidgetUpdateTask(userCompletion) { (dataStore, completion) in
-            let moc = dataStore.viewContext
-            moc.perform {
-                guard let latest = moc.newestGroup(of: .pictureOfTheDay), latest.isForToday else {
```

Comment:
@joewalsh not sure if this will affect anything user-facing but the old POTD code did not consider `siteURL` when fetching `newestGroup`, and the new shared code does.

Reply:
Good catch! It looks like the POTD content group has siteURL set but it's unclear if it'll always match the user's primary language - I've updated it to no longer constraint to siteURL for POTD